.travis.yml

```diff
@@ -13,7 +13,8 @@ matrix:
   fast_finish: true
   allow_failures:
     # allow_failures keyed to skipping tests.
-    - env: SKIP_TESTS="true"
+    - python: 3.5
+      env: SKIP_TESTS="true"
```

Comment:
all the conditions must be met exactly

https://docs.travis-ci.com/user/build-matrix/#matching-jobs-with-allow_failures

you need to complete env